bugbug/models/regressionrange.py

```diff
@@ -74,12 +74,11 @@ def get_labels(self):
 
             bug_id = int(bug_data["id"])
             if "regressionwindow-wanted" in bug_data["keywords"]:
+                classes[bug_id] = 1
+            elif "regressed_by" in bug_data and bug_data["regressed_by"]:
```

Comment:
This could be similified to use only one `if` statement.

Reply:
done